### ISSUE TITLE
Visulization

### ISSUE THREAD

**Issue by dikubab** [weiyithu/SurroundOcc#9]
Please can you provide further guidance on how to visualize the occupancy. I have tried to use the following but not working

python tools/visual.py ./projects/configs/surroundocc/surroundocc.py ckpts/surroundocc.pth --work-dir out/surrocc

FileNotFoundError: [Errno 2] No such file or directory: 'temp/pred.npy'

**Comment by weiyithu** [comment]
You should refer to "Try your own data ,Occupancy prediction" part in README and save the occupancy result. Then you can change the file path as yours in visual.py and visualize the saved result.

**Comment by weiyithu** [comment]
The inferencing .npy and .ply files are used for visualization. We provide two ways for visualization. You can use meshlab to directly visualize .ply files. Or you can change 'temp/pred.npy' as the inferencing .npy path and then run visual.py, which uses mayavi for visualization.  For nuScenes visualization, you can substitute 'inference_infos.pkl' as 'nuscenes_infos_val.pkl'[https://cloud.tsinghua.edu.cn/f/b3f169f4db034764bb87/?dl=1](url).

**Comment by weiyithu** [comment]
We have updated visualization instruction and you can try it. Sorry for the confusion.

**Comment by dikubab** [comment]
The problem is not solved. why don't you provide how to run the visualization code visual.py?

**Comment by weiyithu** [comment]
In fact, we have given an instruction on how to use visual.py here [https://github.com/weiyithu/SurroundOcc/blob/main/docs/run.md](url) . If you have a look at visual.py, you will find it easy to understand and use.

**Comment by dikubab** [comment]
Yeah, I tried it but canot visualize it. It generates an error massage "IsADirectoryError: [Errno 21] Is a directory: './save_dir/'
" it . save_dir is the path for .npy files. 

![image](https://user-images.githubusercontent.com/60455687/228780352-2801c03e-ba9f-4f98-8352-b6021c5bc3ed.png)

**Comment by weiyithu** [comment]
You should use the path of a specific .npy file but not the whole directory. For example, python visual.py ./save_dir/1.npy

**Comment by ryontang** [comment]
> Yeah, I tried it but canot visualize it. It generates an error massage "IsADirectoryError: [Errno 21] Is a directory: './save_dir/' " it . save_dir is the path for .npy files.
> 
> ![image](https://user-images.githubusercontent.com/60455687/228780352-2801c03e-ba9f-4f98-8352-b6021c5bc3ed.png)

hi, what is the version of mayavi?
